### PR TITLE
GitHub Action workflow for builder unit test

### PR DESCRIPTION
**Pull request by ebezzi**
Add a Github Action workflow (a step, really) for running the builder test.

### CODE REVIEW

File: .github/workflows/unittests.yml [hunk 1]
```diff
@@ -25,6 +25,30 @@ jobs:
...
         run: |
           pytest ./api/python/cell_census/tests/
+
+  unit_tests_builder:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.9", "3.10"]
```
Comment (bkmartinjr) [inline .github/workflows/unittests.yml:38]: Tiny, tiny nit:  i'm not sure we need to test on more than one version, as we are the only users.  So strictly from a "how fast is our CI", I suggest a simple test matrix.

**Review by bkmartinjr — APPROVED** [review]
LGTM, with one optional nit you can decide to fix or not :-)

File: .github/workflows/unittests.yml (hunk 1 above)
Comment (ebezzi) [reply .github/workflows/unittests.yml]: I only set 3.10

**Review by atolopko-czi — APPROVED** [review]
👏

**Pull request merged**